packages/z/zsh/files/zprofile

```diff
@@ -1,51 +1,17 @@
+## DO NOT edit this file. Your changes will be overridden on package updates.
+# Instead add your changes to a new profile script under /etc/profile.d/ (create this directory if it does not exist) for system wide use or
+# ~/.zshrc.d for an individual user
+
+# Begin /usr/share/defaults/etc/zprofile
+
 if [ -f /etc/zprofile ]; then
     source /etc/zprofile
 elif [ -f /etc/zsh/zprofile ]; then
     source /etc/zsh/zprofile
 fi
 
-# Source paths that may not be included by systemctl
-if [ -f /usr/share/defaults/etc/profile.d/10-path.sh ]; then
-    emulate sh -c 'source /usr/share/defaults/etc/profile.d/10-path.sh'
-fi
-
-# Source paths not included by systemctl necessary for XDG_DATA_DIRS
-
-if [ -f /usr/share/defaults/etc/profile.d/10-xdg.sh ]; then
-    emulate sh -c 'source /usr/share/defaults/etc/profile.d/10-xdg.sh'
-fi
-
-# Source paths not included by systemctl necessary for flatpaks to be found in the app menu and run
-
-if [ -f /usr/share/defaults/etc/profile.d/70-flatpak.sh ]; then
-    emulate sh -c 'source /usr/share/defaults/etc/profile.d/70-flatpak.sh'
-fi
-
-# Source paths not included by systemctl necessary for snaps to be found in the app menu and run
-
-if [ -f /usr/share/defaults/etc/profile.d/70-snapd.sh ]; then
-    emulate sh -c 'source /usr/share/defaults/etc/profile.d/70-snapd.sh'
-fi
-
-# Source gtk modules variable for menus
-
-if [ -f /usr/share/defaults/etc/profile.d/appmenu-gtk2-module.sh ]; then
-    emulate sh -c 'source /usr/share/defaults/etc/profile.d/appmenu-gtk2-module.sh'
-fi
-
-# Paths for gawk
-
-if [ -f /usr/share/defaults/etc/profile.d/gawk.sh ]; then
-    emulate sh -c 'source /usr/share/defaults/etc/profile.d/gawk.sh'
-fi
-
-# Environment for vte, zsh compatible
-
-if [ -f /usr/share/defaults/etc/profile.d/vte.sh ]; then
-    source /usr/share/defaults/etc/profile.d/vte.sh
-fi
+# This should always be sourced regardless of the above files
```

Comment:
This block should probably be at the top of the script, so the system zprofile in /etc can override vendor config settings...?

Reply:
Good point. Fixed.